highfidelity/hifi
# WL 21666 - Window.domainChanged() isn't emitted if domain isn't running

> The address in the Interface title bar changes but the signal isn't emitted.
> 
> function onDomainChanged(domain) {
> print("Domain changed: " + domain);
> }
> 
> Window.domainChanged.connect(onDomainChanged);
> 
> Task
> -------
> Make sure that if a domain is changed to a domain that isn't running that it still gives back a signal.
> Worklist [21666](https://worklist.net/21666)

## Test Plan
1. Install this PR, and enable Advanced/Developer Menus
2. Open the Console and run: `Window.domainChanged.connect(function(e) { print("Domain Changed: " + e)});`
2. Start connecting back and forth from different domains, confirm that a new console entry fires when you _disconnect_ from a domain as well as _connect_.
